### PR TITLE
Setuptypescriptl

### PR DESCRIPTION
**Pull request by veroggu**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+directories:
+- node_modules
+notifications:
+email: false
+branches:
+except:
+- "/^v\\d+\\.\\d+\\.\\d+$/"
+script:
+  - npm test
+
```
Comment (holgiosalos) [inline .travis.yml]: remove whitespaces on line 12 without deleting that line

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+              options: {
+                transpileOnly: true,
+              },
+            },
+          ],
+        },
+      ],
+    },
+  }
+
```
Comment (holgiosalos) [inline webpack.config.js]: remove whitespaces on line 25 without deleting that line.

**Review by holgiosalos — COMMENTED** [review]
Please apply code review changes on this branch and do not create more pull request until this PR gets approved and merged.

File: .travis.yml (hunk 1 above)
Comment (veroggu) [reply .travis.yml]: done!

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**